lib/ansible/modules/cloud/amazon/batch_compute_environment.py

```diff
@@ -0,0 +1,496 @@
+#!/usr/bin/python
+# (c) 2017, Jon Meran <jonathan.meran@sonos.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: batch_compute_environment
```

Comment:
Can you please prefix this (and the others) with `aws_` since `batch_compute_environment` might be confused with some other batching system, and we have just one flat module namespace.

Reply:
fixed